### PR TITLE
Fire select event after boxend

### PR DESCRIPTION
**Pull request by Sonia-96**
### Description
Fire select event in the callback of `boxend` event, so the name attribute will pop up automatically.

(Click Play button to see the gif)

Screen Recording 2024-11-27 at 10 19 14 AM

### Quality Checks
 - [x] New code is 100% tested
 - [x] Code has been formated
 - [x] Code has been linted
 - [x] Docstrings for new methods have been added

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ channels:
   - conda-forge
 
 dependencies:
-  - python>=3.8
+  - python=3.10
```
Comment (swainn) [inline environment.yml:16]: @Sonia-96 please revert this change

**Review by swainn — CHANGES_REQUESTED** [review]

**Review by swainn — APPROVED** [review]

**Review by sdc50 — APPROVED** [review]
